packages/bot-framework/README.md

```diff
@@ -0,0 +1,18 @@
+# foo
```

Comment:
Imagine I want to write a new bot.*  Please describe the list of services provided by this library and why I would want to use them.

* I know we said we never want to write a new bot, but I think imagining we might will result in a README that better helps maintainers.